.gitignore

```diff
@@ -26,6 +26,7 @@ src/*.so.*
 *.vcxproj*
 *.tmp
 *.sln
+/src/plugins/smith_waterman/vc*.pdb
```

Comment:
What file is this about? As far as I know, the all ".pdb" files should be moved to the build directory

Reply:
This file is generated in ugene\src\plugins\smith_waterman\_tmp\obj\debug\vc142.pdb